uber/custom_tags.py

```diff
@@ -182,8 +182,8 @@ def render(self, context):
                 val = val.strftime(c.TIMESTAMP_FORMAT)
             else:
                 selected = 'selected="selected"' if str(val) == str(default) else ''
-            val  = str(val).replace('"',  '&quot;').replace('\n', '')
-            desc = str(desc).replace('"', '&quot;').replace('\n', '')
+            val  = html.escape(str(val), quote=False).replace('"',  '&quot;').replace('\n', '')
+            desc = html.escape(str(desc), quote=False).replace('"', '&quot;').replace('\n', '')
```

Comment:
I realize that it's a little silly to say `quote=False` for the escape method and then manually replace quotes, but I wasn't sure if there was a specific reason we didn't want to replace single quotes. I wanted to keep this as close to original functionality as possible, since it will affect a lot of different places and I don't trust myself to test them all.